specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json

```diff
@@ -0,0 +1,286 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "KeyVaultClient",
+    "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
+    "version": "7.2-preview"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/backup": {
+      "post": {
+        "tags": [
+          "FullBackup"
+        ],
+        "operationId": "FullBackup",
+        "description": "Creates full backup using the user provided SAS token to azure blob storage.",
```

Comment:
Would it be more accurate to call it Azure Blob storage container? These description strings become part of REST API reference doc, so I would insist on accurate names and also capitalization.